### ISSUE TITLE
[BITV] 9.1.4.11/4.3 - Media files marked as selected - The checkboxes do not meet the contrast requirements for graphical informative elements depending on the background (1)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/photos#2160]
**Blocked by** 
* https://github.com/nextcloud/server/issues/42121

as the contrast ratio with the background is below 3:1. It is recommended to place these elements outside of images. The issue also affects unselected checkboxes (see screenshot)

![image](https://github.com/nextcloud/server/assets/6078378/f51dc136-f8ac-4550-9484-3f0a1364c2c1)
![image](https://github.com/nextcloud/server/assets/6078378/ec21112a-c974-4c35-87d7-9dc9e665d4f6)

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-3dbfb7615a-v4-n3

**Comment by susnux** [comment]
Idea: add some blur background to ensure contrast, similar to the dashboard widgets

**Comment by JuliaKirschenheuter** [comment]
@artonge could you please help us with this ticket?

**Comment by JuliaKirschenheuter** [comment]
Thank you @artonge!

"Unchecked & Picture hovered" and "Checked" still remaining not accessible ;(

I would suggest not to have special hover styles, because they are not relevant for a11y and leave everything only on checked and unchecked state, like:

Before | After
-- | --
Unchecked and Checkbox hovered | ![Screenshot from 2023-12-06 15-37-07](https://github.com/nextcloud/photos/assets/6653109/9abd704c-f87d-4f4d-ad5d-bb7f595fc833)
Checked and Checkbox hovered | ![Screenshot from 2023-12-06 15-37-02](https://github.com/nextcloud/photos/assets/6653109/c372e368-5474-428b-b3e5-24a3c515a531)

Focus state have to have a double outline (black and white) like on most buttons:

![Screenshot from 2023-12-13 17-20-18](https://github.com/nextcloud/photos/assets/6078378/fc3d81c2-dff8-4a56-aca0-6c4f53ee81f3)


In light theme background have to be not dark but light like:

![Screenshot from 2023-12-13 17-16-45](https://github.com/nextcloud/photos/assets/6078378/bbb60fb0-1934-4300-bfb9-1f340c0a8141)

@nimishavijay would you approve such solution?

**Comment by artonge** [comment]
> I would suggest not to have special hover styles, because they are not relevant for a11y and leave everything only on checked and unchecked state, like:

That's what I ended up doing :), but I thought I updated the description :thinking:

**Comment by nimishavijay** [comment]
Sounds good to me as well. So when you hover over the picture, the checkbox with the background appears, and when you hover over the checkbox, it shows the normal hover feedback for a checkbox component. Would that work? :)
